lib/pages/withoutProps/setup.js

```diff
@@ -14,6 +15,17 @@ const setup = (publishPath) => {
   pages.forEach(({ filePath }) => {
     logItem(filePath);
 
+    // HACK: If i18n, Next internals expect a 404.html at the top level of the publish directory
+    // TO-DO: investigate framework and why this isn't an issue on Vercel and only certain routes
```

Comment:
1. isn't it technically both netlify and next? because in non-i18n, next puts it top level but doesnt in i18n even though it should be there (or in other words, expects it there).

2. i'll open an issue for that second part. at this point it can't be a blocker for the PR but we can/should definitely discuss what may be better going forward

Reply:
https://github.com/netlify/next-on-netlify/issues/129